zegl/kube-score
# chore: package as nix flake

```
RELNOTE: make kube-score discoverable as nix flake
```

You see a comment for the v1.7.2 tag, unless you'd not like to add it and force push this tag with that vendor sha, then let's call it supported from now on.

How to test it:

- check out this pr
- have nix installed [`curl -L https://nixos.org/nix/install | sh`](https://nixos.org/download.html)
- install nix flakes pre release: `nix-env -iA nixpkgs.nixFlakes`
- `nix --experimental-features 'nix-command flakes' shell  -c kube-score version`
- `nix --experimental-features 'nix-command flakes' shell  -c which kube-score`

Also interesting to check:

- `nix flake show github:blaggacao/kube-score`
- `nix shell github:blaggacao/kube-score` (as a temporary solution for the inclined visitor out of the nix community until this is - maybe - merged. Be warned I'll not rebase or update my fork.)
- in the new shell where you are dropped in by the above command: `which kube-score`
